windows_10_docker.json

```diff
@@ -1,5 +1,34 @@
 {
   "builders": [
+    {
+      "boot_wait": "6m",
+      "communicator": "winrm",
+      "cpu": "2",
+      "disk_size": "{{user `disk_size`}}",
+      "floppy_files": [
+        "{{user `autounattend`}}",
+        "./floppy/WindowsPowershell.lnk",
+        "./floppy/PinTo10.exe",
+        "./scripts/fixnetwork.ps1",
+        "./scripts/disable-screensaver.ps1",
+        "./scripts/disable-winrm.ps1",
+        "./scripts/enable-winrm.ps1",
+        "./scripts/microsoft-updates.bat",
+        "./scripts/win-updates.ps1"
+      ],
+      "guest_additions_mode": "none",
+      "iso_checksum": "{{user `iso_checksum`}}",
+      "iso_checksum_type": "{{user `iso_checksum_type`}}",
+      "iso_url": "{{user `iso_url`}}",
+      "ram_size": "2048",
+      "shutdown_command": "shutdown /s /t 10 /f /d p:4:1 /c \"Packer Shutdown\"",
+      "switch_name": "{{user `switch_name`}}",
+      "type": "hyperv-iso",
```

Comment:
I have started the windows_10_docker.json template not from ISO, but from a previously built box. In my case it was vmware-vmx instead of vmware-iso.
For Hyper-V it should be hyperv-vmcx instead of hyperv-iso ( https://www.packer.io/docs/builders/hyperv-vmcx.html ) so you don't have to add all the basic provision scripts.